docs/fr/index.md

```diff
@@ -1,153 +1,143 @@
-# Démarrer sur l'espace de travail d'analyse avancée
+# Bienvenue dans l'espace de travail Advanced Analytics
 
-Le
-**[portail de l'espace de travail d'analyse avancée](https://kubeflow.aaw.cloud.statcan.ca/)**
-est un excellent endroit où explorer les ressources dont il sera question ici,
-et y accéder.
+<center>
+![Statistiques](images/statistiques-sur-la-lune.jpg)
+</center>
 
-### Que cherchez-vous ?
+## La documentation de l'espace de travail d'analyse avancée
 
-[![Démarrage](images/Commencer.PNG)](#démarrer-avec-l'aw)
+_Bienvenue dans le monde de la science des données et de l'apprentissage automatique !_
 
-[![Analyse](images/Analyse.PNG)](#experiences)
-[![Publication](images/Publication.PNG)](#publication)
-[![Pipelines](images/Pipelines.PNG)](#pipelines)
-[![Collaboration](images/Collaboration.PNG)](#collaboration)
+<!-- prettier-ignore -->
+!!! info "Qu'est-ce que l'AAW?"
+     **[Advanced Analytics Workspace](https://analytics-platform.statcan.gc.ca/)** est une plateforme open source conçue pour les data scientists, les data stewards, les analystes et les chercheurs familiarisés avec les outils open source et le codage. Développé par des scientifiques de données pour des scientifiques de données, AAW fournit un environnement flexible qui permet aux praticiens avancés de faire leur travail en toute simplicité.
```

Comment:
Change "Advanced Analytics Workspace" to Espace de travail d'analyse avancée
Change "scientifiques des données" to data scientists
Change AAW to ETAA
stewards is not french... but cant find the correct translation.

Reply:
How does les intendants des données sound?